README.md

```diff
@@ -24,13 +24,13 @@ Then using the property $(\log_{b} a) = (\frac{1}{(\log_{k} b)})(\log_{k} a)$ wh
 
 $T(n) \leq c(\frac{1}{(\log_{5} 2)})(\log_{5} n)$, $T(n) \leq c(\frac{1}{(\log_{2} 5)})(\log_{2} n)$
 
-We can now let $c(\frac{1}{(\log_{5} 2)})$ and $c(\frac{1}{(\log_{2} 5)})$ = g and h respectively, where g and h are both constants.
+We can now let $c(\frac{1}{(\log_{5} 2)})$ and $c(\frac{1}{(\log_{2} 5)})$ both equal $c$, which means that $c$ can be any constant;
```

Comment:
Still not there. First, your notation $c()$ seems to suggest that $c$ is a function, which it isn't -- if you want to set $c$ equal to a value, then please do that. And while $c$ can be any constant, it cannot be two different  values at the same time.